lib/automations/update-milestone/get-config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+const { setFailed, getInput: coreGetInput } = require( '@actions/core' );
+
+const inputs = {
+	targetMilestone: {
+		input: 'target_milestone',
+		default: '',
+		required: false,
+	},
+};
+
+const getInput = ( input ) => {
+	const value = coreGetInput( input.input ) || input.default;
+	if ( input.required && ! value ) {
+		throw new Error(
+			`Update Milestone: Missing required input ${ input.input } your input: ${ input }`
```

Comment:
Are there any dangers to outputting the user input?

Reply:
@mikejolley That's a very good point. I'm not sure about it but I would like to learn the consequence of it. I'll appreciate it if you can share more details. 

I'll commit the below change for now: 
```suggestion
			`Update Milestone: Missing required input ${ input.input }`
```